x-pack/plugins/security_solution/public/flyout/document_details/right/components/description.tsx

```diff
@@ -42,11 +42,9 @@ export const Description: FC = () => {
         indexName,
         scopeId,
         banner: {
-          title: (
-            <FormattedMessage
-              id="xpack.securitySolution.flyout.right.about.description.rulePreviewTitle"
-              defaultMessage="Preview rule details"
-            />
+          title: i18n.translate(
```

Comment:
just curious on why making this change?

Reply:
@PhilippeOberti previous appraoach would result in non-serializable value (react element)